frontend/summary/dataPivot/DataPivotExtension.js

```diff
@@ -110,7 +112,7 @@ _.extend(DataPivotExtension, {
             _dpe_name: "experiment",
             _dpe_key: "experiment id",
             _dpe_cls: Experiment,
-            _dpe_option_txt: "Show experiment",
+            _dpe_option_txt: "Show animal experiment",
```

Comment:
updated these descriptions to add additional context, which isn't needed when autopopulated but was needed in the new table